app.js

```diff
@@ -10,6 +31,141 @@ function Book(title, author, pages, read) {
     }
 }
 
-myBook = new Book('Lord of the Rings', 'Tolkien', 365, true);
+function createBook(book) {
+  myLibrary.push(book);
+  document.querySelector('.input-form').classList.toggle('form-visibility');
+  localStorage.setItem("myLibrary", JSON.stringify(myLibrary));
+  render(myLibrary);
+}
+
+// Reads input values from the form.
+function readInputValues(){
+  return {
+  bookTitle: document.getElementById("book-title").value,
+  authorName: document.getElementById("author-name").value,
+  description: document.getElementById("book-description").value,
+  pagesNum: document.getElementById("book-pages").value,
+  status: document.querySelector('input[name="read"]:checked').value
+  }
+}
+function addBookToLibrary() {
+  let bookInfo = readInputValues();
+  const id = randomId();
+  let book =new Book(id, bookInfo.bookTitle, bookInfo.authorName, bookInfo.description, bookInfo.pagesNum, bookInfo.status);
+  let valid = inputValid(bookInfo.bookTitle, bookInfo.authorName, bookInfo.description, bookInfo.pagesNum);
+
+  if (valid == true) {createBook(book);}
+  clearForm();
+}
+
+
+function clearForm()
+{
+  form.reset();
+  operation = null;
+}
+// function for handling click event on New Form.
+toggleForm(formBtn);
+
+function toggleForm(button) {
+  button.addEventListener('click', function(){
+    form.classList.toggle('form-visibility');
+  });
+}
+
+// function for handling click event on Submit.
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history, your code is clear enough to understand 🙂 ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))